packages/office-ui-fabric-react/src/components/ActivityItem/ActivityItemPage.tsx

```diff
@@ -1,74 +1,43 @@
 import * as React from 'react';
-import {
-  ExampleCard,
-  ComponentPage,
-  IComponentDemoPageProps,
-  PropertiesTableSet,
-  PageMarkdown
-} from '@uifabric/example-app-base';
-import { ComponentStatus } from '../../demo/ComponentStatus/ComponentStatus';
 import { ActivityItemStatus } from './ActivityItem.checklist';
 import { ActivityItemBasicExample } from './examples/ActivityItem.Basic.Example';
 import { ActivityItemPersonaExample } from './examples/ActivityItem.Persona.Example';
 import { ActivityItemCompactExample } from './examples/ActivityItem.Compact.Example';
+import { IDemoPageProps } from '../../demo/components/IDemoPageProps';
+import { DemoPage } from 'office-ui-fabric-react/lib/demo/components/DemoPage';
```

Comment:
>office-ui-fabric-react/lib/demo [](start = 26, length = 31)

Did you mean to do a relative path import from here?

Reply:
Yes, I did actually mean to do relative.